package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@moxy/next-webpack-oneof",
+  "version": "0.0.0",
+  "description": "Wraps all webpack loader rules into a single oneOf rule.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --coverage",
+    "lint": "eslint .",
+    "prerelease": "npm t && npm run lint",
+    "release": "standard-version",
+    "postrelease": "git push --follow-tags origin master && npm publish"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moxystudio/next-webpack-oneof.git"
+  },
+  "files": [
```

Comment:
Perhaps it's better to  make it an empty array? You may use `npm pack --dry-run` to test what will be published.